package.json

```diff
@@ -127,7 +127,7 @@
     "file-loader": "^0.11.1",
     "fs-promise": "^2.0.2",
     "glob": "^7.1.1",
-    "html-webpack-harddisk-plugin": "^0.1.0",
+    "html-webpack-harddisk-plugin": "^2.0.0",
```

Comment:
# Upgrade `html-webpack-harddisk-plugin` to the latest version (2.0.0) 
```
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning " > uglifyjs-webpack-plugin@1.3.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
```

Running `yarn-dev-ee` after this produces the following error:
```
[cljs] ...[webpack-cli] TypeError: compiler.plugin is not a function
[frontend]     at BannerWebpackPlugin.apply (/Users/offbynone/code/work/metabase/node_modules/banner-webpack-plugin/index.js:16:14)
[frontend]     at createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:74:12)
[frontend]     at create (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:123:16)
[frontend]     at webpack (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:147:32)
[frontend]     at WebpackCLI.f [as webpack] (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/index.js:54:15)
[frontend]     at WebpackCLI.createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1845:29)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/@webpack-cli/serve/lib/index.js:81:30)
[frontend]     at async Promise.all (index 1)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1289:13)
error Command failed with exit code 2.
```

[BannerWebpackPlugin](https://github.com/lcxfs1991/banner-webpack-plugin) we're using was last updated on 25 Sep 2017. Webpack seems to have the their own [banner plugin](https://webpack.js.org/plugins/banner-plugin). Not quite sure why we chose to go with the 3rd party version. Probably because it allows to specify chunks individually.